src/main.ts

```diff
@@ -20,6 +21,20 @@ library.add(fas, far)
 loadFonts()
 
 const app = createApp(App)
+
+// Only track usage statistics if the user has not opted out
+if (window.localStorage.getItem('cockpit-enable-usage-statistics-telemetry')) {
+  Sentry.init({
+    app,
+    dsn: 'https://5176a29493da4d31a6384c55d291cd3d@o4507692193415168.ingest.us.sentry.io/4507692196036608',
+    integrations: [Sentry.browserTracingIntegration(), Sentry.replayIntegration()],
+    tracesSampleRate: 1.0, //  Capture 100% of the transactions
+    tracePropagationTargets: [],
+    replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
```

Comment:
never used the replay options ( only available in paid ) but I think itś supposed to generate a "video-like" session of users using the app, looking forward to see it in action :)

Reply:
I'm also curious 👀